cassandra/Ballerina.toml

```diff
@@ -1,13 +1,13 @@
 [project]
 org-name="ballerinax"
-version="0.8.6"
+version="0.9.0"
```

Comment:
Shall we change the version to 0.99.0 please. Since the current released version is 0.8.6, we need to keep 0.9.0 for 1.2.x release path

Reply:
Updated in bf1bea8